doc_src/cmds/echo.rst

```diff
@@ -57,18 +57,20 @@ If ``-e`` is used, the following sequences are recognized:
 Example
 -------
 
-
-
 ::
 
     echo 'Hello World'
 
 Print hello world to stdout
 
-
-
 ::
 
     echo -e 'Top\\nBottom'
 
 Print Top and Bottom on separate lines, using an escape sequence
+
+
+See Also
+--------
+
+- :ref:`printf <cmd-printf>` command
```

Comment:
I'd leave a short explanation as to why it's related or why you should look it up.

E.g. "the printf command, for more control over output formatting"?